src/main/java/org/fcrepo/client/TransactionBuilder.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree.
+ */
+package org.fcrepo.client;
+
+import java.net.URI;
+
+import org.apache.http.client.methods.HttpRequestBase;
+import org.apache.http.util.Args;
+
+/**
+ * Creates different RequestBuilders for interacting with the Fedora Transaction API.
+ *
+ * @author mikejritter
+ */
+public class TransactionBuilder {
+
+    public static final String TRANSACTION_ENDPOINT = "fcr:tx";
+
+    private final URI uri;
+    private final FcrepoClient client;
+
+    /**
+     * Instantiate builder. Throws an IllegalArgumentException if either the uri or client are null.
+     *
+     * @param uri    uri of the resource this request is being made to
```

Comment:
Maybe clarify that this should either be a txUri, the tx endpoint, or the repository root? (basically that it shouldn't be a URI to an actual resource)